tests/test_views_item_patch.py

```diff
@@ -57,34 +58,47 @@ def test_valid_patch(app, test_records, test_patch, content_type):
         assert res.status_code == 200
 
         # Check that year changed.
-        assert previous_year != get_json(client.get(url))['metadata']['year']
+        new_year = get_json(client.get(url))['metadata']['year']
+        assert previous_year != new_year
+        es.indices.flush('*')
```

Comment:
Should we use [`flush_and_refresh`](https://github.com/inveniosoftware/invenio-search/blob/ebe51c79a7dde1b0b2f491935f7a4339f91c4282/invenio_search/ext.py#L176) instead?  since it is taking care of more than `flush` and also would wait if indices could not be indexed straight away.

Reply:
Good idea